op-service/metrics/event.go

```diff
@@ -33,6 +33,21 @@ func NewEvent(factory Factory, ns string, subsystem string, name string, display
 	}
 }
 
+func NewEventLight(factory Factory, ns string, name string, displayName string) *Event {
```

Comment:
Go will default an uninitialized string to `""`, so this could delegate to `NewEvent(factory,ns, "", name, displayName)`.

I'm thinking about if we want to remove this function & have callers be responsible for passing in the empty subsystem string (several already do this).

The other option is to rename `NewEvent` to be `NewEventWithSubsystem` and this to be `NewEvent`. The transaction manager is the only place the sub-system label is applied.

Reply:
updated, plz check again :)